WebBookStore/Controllers/CartitemController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Org.BouncyCastle.Crypto.Macs;
+using WebBookStore.Dto;
+using WebBookStore.Interfaces;
+using WebBookStore.Models;
+using WebBookStore.Repository;
+
+namespace WebBookStore.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CartitemController : ControllerBase
+    {
+        private readonly ICartitemRepository _cartitemRepository;
+        private readonly IMapper _mapper;
+
+        public CartitemController(ICartitemRepository cartitemRepository, IMapper mapper) 
+        {
+            _cartitemRepository = cartitemRepository;
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        public IActionResult GetCartitems()
+        {
+            var cartitems = _mapper.Map<List<Cartitem>>(_cartitemRepository.GetCartitems());
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return Ok(cartitems);
```

Comment:
thừa dòng + sử dụng toán tử 3 ngôi cho trường hợp if dodnw giản và 2 cái return sát nhau